app/assets/stylesheets/application.scss

```diff
@@ -17,6 +17,6 @@
 @import "bootstrap-sprockets";
 // We override bootstrap variables in a file a subdirectory
 // so that it's not pulled in twice with the @import "*" below
-@import "bootstrap-overrides/bootstrap-overrides";
 @import "bootstrap";
+@import "bootstrap-overrides/bootstrap-overrides";
```

Comment:
https://github.com/twbs/bootstrap-sass#sass-1 says this needs to be before `@import "bootstrap"`.  Otherwise it won't work for overriding bootstrap variables.  See also my comment below in "bootstrap-overrides.scss".